### PR TITLE
[Workers] Add documentation for cf-tutor tool

### PR DESCRIPTION
**Pull request by db-cloudflare**
### Summary

Adding documentation for a beta release of an application that provides a guided learning experience for newer developers.

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/workers/get-started/cf-tutor.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: CLI guided learning (beta)
+pcx_content_type: get-started
+sidebar:
+  order: 2
+  group:
+    badge: Beta
```
Comment (db-cloudflare) [inline src/content/docs/workers/get-started/cf-tutor.mdx]: Still working on getting the beta tag to render properly.

File: src/content/docs/workers/get-started/cf-tutor.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  badge:
+    text: Beta
+head:
+  - tag: title
+    content: Get started - CLI guided learning
+---
+
+import { Badge, Render } from "~/components";
+
+Cloudflare CLI tutor, or `cf-tutor` is a guided learning experience within your terminal. If you are unfamiliar with Workers or are new to command line interface (CLI) based development, cf-tutor offers introductions on how to use Workers with other Cloudflare products through your terminal.
```
Comment (Oxyjun) [inline src/content/docs/workers/get-started/cf-tutor.mdx]:
```suggestion
Cloudflare CLI tutor, or `cf-tutor`, is a guided learning experience within your terminal. If you are unfamiliar with Workers, or are new to command line interface (CLI) based development, cf-tutor offers an introduction on how to use Workers with other Cloudflare products through your terminal.
```

File: src/content/docs/workers/get-started/cf-tutor.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`cf-tutor` is currently a beta application in pre-release. To download the application, you can use the `git clone` in your terminal or download the application from the releases page on the project repo.
```
Comment (Oxyjun) [inline src/content/docs/workers/get-started/cf-tutor.mdx]:
```suggestion
`cf-tutor` is currently a beta application in pre-release. To download the application, you can use the `git clone` in your terminal, or download the application from the releases page on the project repo.
```

File: src/content/partials/workers/cf-tutor.mdx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+3. Run the command `git clone git@github.com:db-cloudflare/cf-tutor.git` and enter your SSH key to download cf-tutor.
+4. Run the command `cd cf-tutor` to enter the application folder.
+5. Run the command `npm i` to install all dependendant packages.
+6. Once the dependencies have installed, run `npm start ` to run the application
+
+</TabItem>
+
+<TabItem label="Download from browser">
+
+1. Open [cf-tutor releases](https://github.com/db-cloudflare/cf-tutor/releases)
```
Comment (Oxyjun) [inline src/content/partials/workers/cf-tutor.mdx]:
```suggestion
1. Open [cf-tutor releases](https://github.com/db-cloudflare/cf-tutor/releases).
```

File: src/content/partials/workers/cf-tutor.mdx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+2. Download the cf-tutor.zip file and extract the folder inside it into your desired location.
+3. In your terminal, navigate to the cf-tutor folder. You can change directory by using the `cd` command followed by the folder path. You can either enter the full path such as `cd Documents/Projects/cf-tutor` or you can use `cd` multiple times to go folder by folder. To see the files and folders in the directory you are currently in, run `ls`.
+4. Run the command `npm i` to install all dependendant packages.
+5. Once the dependencies have installed, run `npm start ` to run the application
```
Comment (Oxyjun) [inline src/content/partials/workers/cf-tutor.mdx]:
```suggestion
5. Once the dependencies have installed, run `npm start ` to run the application.
```

**Review by Oxyjun — APPROVED** [review]

**Pull request merged**